apps/website/src/icons/IconGithub.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+type IconProps = {
+  size?: number;
+  className?: string;
+};
+
+const IconGithub: React.FC<IconProps> = ({ size = 24, className }) => (
```

Comment:
```suggestion
const IconGitHub: React.FC<IconProps> = ({ size = 24, className }) => (
```
Super nitpick, GitHub's brand name has a capital `H` (would also apply to the file name)